pep-0458-tuf-online-keys.rst

```diff
@@ -553,18 +561,19 @@ recommends diversity of keys for certain applications.
 Number Of Keys Recommended
 --------------------------
 
-The *timestamp*, *snapshot*, and *bins* roles require continuous delivery.
-Even though their respective keys MUST be online, this PEP requires that the
-keys be independent of each other.  Different keys for online roles allow for
-each of the keys to be placed on separate servers if need be, and prevents side
-channel attacks that compromise one key from automatically compromising the
-rest of the keys.  Therefore, each of the *timestamp*, *snapshot*, and *bins*
-roles MUST require (1, 1) keys.
+The *timestamp*, *snapshot*, and *pypi-signed* roles require continuous
+delivery.  Even though their respective keys MUST be on-pypi, this PEP requires
+that the keys be independent of each other.  Different keys for pypi-signed
+roles allow for each of the keys to be placed on separate servers if need be,
+and prevents side channel attacks that compromise one key from automatically
```

Comment:
568: seems odd "compromise one key from automatically compromising the rest of the keys"
I think the first compromise should be replaced with "prevent"